kodilla-advanced-tests/src/test/java/com/kodilla/parametrized_tests/homework/GamblingMachineTestSuite.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.parametrized_tests.homework;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class GamblingMachineTestSuite {
+
+    private final GamblingMachine testee = new GamblingMachine();
+
+    @Test
+    public void shouldReturnExceptionWhenUserNumbersIsEmpty() {
+        assertThrows(InvalidNumbersException.class, () -> testee.howManyWins(new HashSet<>()));
+    }
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/exceptionCorrectSetOfNumbers.csv")
+    public void shouldReturnTrueWhenSetOfNumbersIsCorrectForUserNumber(String userNumbers, int expected) throws InvalidNumbersException {
```

Comment:
Testujesz metodę howManyWins, tak? To czemu w nazwie metody testującej jest should return true? Nie rozumiem. Ta metoda nie powinna poprostu rzucić wyjątku, kiedy w pliku jest w linijce 6 liczb.